### DISCUSSION TITLE
How to set prediction confidence threshold

### DISCUSSION THREAD

**Discussion by Pranay-Pandey** [ultralytics/ultralytics#678] [Q&A] [answered]
How to set the prediction confidence threshold for the prediction task in python?

**Comment by AyushExel** [comment, marked as answer]
conf=value

**Reply by mochiao** [reply]
May I ask where to change this parameter? in result()?

**Reply by Pranay-Pandey** [reply]
YOLO Model Object.

Example usage - 
model = YOLO("yolov8n.pt")
results = model(file, **conf=value**)

**Reply by glenn-jocher** [reply]
@Pranay-Pandey to set the prediction confidence threshold when using a YOLOv8 model in Python, you can adjust the `conf` parameter directly when calling the model on your data. Here's how you can do it:

```python
model = YOLO('yolov8n.pt')  # Load the model
results = model(file, conf=0.25)  # Set your desired confidence threshold
```

Replace `0.25` with your desired confidence threshold value. This will filter out predictions below the specified confidence level.
